api/public_api_v1.yaml

```diff
@@ -35,6 +35,37 @@ paths:
         '404':
           description: No service found for the provided ID.
       parameters: []
+  "/services":
+    get:
+      operationId: getServices
```

Comment:
this operation is more specific, like `getServicesByRecipient`

Reply:
according to the swagger 2.0 specs there's no way to map multiple operations on the same endpoint which differs only by a query string parameter. if you want to name this getServicesByRecipient then a /services-by-recipients endpoint would be a better fit

Reply:
I prefer to keep the resource as is, thanks for checking